src/psyclone/tests/dependency_test.py

```diff
@@ -866,7 +905,7 @@ def test_lfric_acc_operator():
     create_acc_arg_list.generate(var_accesses=var_accesses)
     var_info = str(var_accesses)
     assert "lma_op1_proxy%ncell_3d: READ" in var_info
-    assert "lma_op1_proxy%local_stencil: WRITE" in var_info
+    assert "lma_op1_proxy%local_stencil: READ" in var_info
 
```

Comment:
Please could you add a check here that the CMA argument has WRITE access.

Reply:
Done.